doc/source/guide/merging.rst

```diff
@@ -42,6 +42,13 @@ With the :option:`--merge-mode-functions` you can change this:
 - ``merge-use-line-max``: Allow same function on different lines, in this case use maximum line.
 - ``separate``: Allow same function on different lines. Instead of merging keep the functions separate.
 
+If you have the mangled or demangled name defined only in some files (mainly if you user cross compiling
+with ``gcc-14`` and older ``gcc`` versions) the mangled names can be defined only in one input file.
+With the :option:`--merge-mode-function-names` you can change this:
```

Comment:
`... change this behavior:

Reply:
> `... change this behavior:

With this button you can add a suggestion:
![image](https://github.com/user-attachments/assets/76b91fbe-5c1d-4bfe-baf2-eb3844339e8f)

```suggestion
With the :option:`--merge-mode-function-names` you can change this behaviour:
```